app/views/components/upload/YouHaveUploadedFilesPage.scala.html

```diff
@@ -0,0 +1,29 @@
+@import config.AppConfig
+@import helpers.UpscanMessageHelper.getPluralOrSingular
+
+@this(layout: Layout,
+        formHelper: FormWithCSRF,
+        govukButton: GovukButton,
+        p: components.p,
+        radios: components.radios)
+
+@(form: Form[_], radioOptions: Seq[RadioItem], postAction: Call, filesUploaded: Int)(implicit request: UserRequest[_], messages: Messages, appConfig: AppConfig)
+
+@pageHeadingAndTitle = @{getPluralOrSingular(filesUploaded, filesUploaded)("youHaveUploadedFiles.headingAndTitle", "youHaveUploadedFiles.headingAndTitle").toString()}
```

Comment:
The message key is duplicated here, the first parameter should be `"youHaveUploadedFile.headingAndTitle"`
```suggestion
@pageHeadingAndTitle = @{getPluralOrSingular(filesUploaded, filesUploaded)("youHaveUploadedFile.headingAndTitle", "youHaveUploadedFiles.headingAndTitle").toString()}
```

Reply:
![image](https://user-images.githubusercontent.com/53828896/140046678-c6bcb5e2-d451-480c-a50b-a540e9a5c01e.png)

Reply:
Removed the duplicate one.